src/ExistingAttributesPersistence.php

```diff
@@ -18,8 +18,11 @@ public static function bootExistingAttributesPersistence()
     {
         // Register once for all, the existing attributes of the table
         $dummy = new static;
-        static::$ExistingAttributesPersistence_tableColumns = \DB::connection()->getSchemaBuilder()->getColumnListing($dummy->getTable());
```

Comment:
The bootTRAITNAME() method should be called only once per model, so caching table columns here should not have an impact. Do you have some profling or log to confirm?